example2/bin/tokens/README.md

```diff
@@ -0,0 +1,23 @@
+These figma files were generated using https://tokenzengarden.design/explore/e70c7218-8978-4254-967f-4e119219ce5c
+
+# Manual changes
+
+The following manual changes were applied to the files to make them work with the current parsers.
+
+# Remove all non !/-* math operators
+Remove all function calls inside the values in the json.  Function calls are not supported at this time.
+
+Remove the `roundTo` and `^` operators and the associated parenthesis.
+
```

Comment:
I left example 2 in place because example 1 is pretty simple with a standard app and thought it would be good to have two example token sets.  Yes, the example 2 tokens should be part of the test suite but I wasn't sure how you wanted to structure tests that read from files.  Currently, none of your tests read from files AFAIK.